forge-gui/res/cardsfolder/upcoming/lara_croft_tomb_raider.txt

```diff
@@ -0,0 +1,19 @@
+Name:Lara Croft, Tomb Raider
+ManaCost:G U R
+Types:Legendary Creature Human Ranger
+PT:3/4
+K:First Strike
+K:Reach
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ Whenever CARDNAME attacks, exile up to one target legendary artifact card or legendary land card from a graveyard and put a discovery counter on it. You may play a card from exile with a discovery counter on it this turn.
+SVar:TrigExile:DB$ ChangeZone | ValidTgts$ Artifact.Legendary,Land.Legendary | Origin$ Graveyard | TargetMin$ 0 | TargetMax$ 1 | TgtPrompt$ Select up to one target legendary artifact card or legendary land card from a graveyard | Destination$ Exile | RememberChanged$ True | SubAbility$ DBPutCounter
```

Comment:
Maybe just WithCountersType and then you wouldn't need remembered or lines 9-10?

Reply:
it's not the same rules wise

Reply:
Ah I see "and put" vs. "with a"